### PR TITLE
Proofread UPGRADE guide

### PR DESCRIPTION
**Pull request by wouterj**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]

**Review by maxbeckers — APPROVED** [review]
Thanks for that :)

File: src/Symfony/Component/Cache/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ CHANGELOG
 7.2
 ---
 
- * `igbinary_serialize()` is not used by default when the igbinary extension is installed
+ * `igbinary_serialize()` is no longer used instead of `serialize()` by default when the igbinary extension is installed,
+   due to behavior compatibilities between the two
```
Comment (xabbuh) [inline src/Symfony/Component/Cache/CHANGELOG.md:8]:
```suggestion
   due to behavior incompatibilities between the two
```

Shouldn't it be like this?

File: src/Symfony/Component/Mailer/CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ CHANGELOG
    you now need to use the `IncompleteDsnTestTrait`.
 
  * Make `TransportFactoryTestCase` compatible with PHPUnit 10+
- * Support unicode email addresses such as "dømi@dømi.fo"
+ * Support unicode email addresses such as "dømi@example.org"
```
Comment (xabbuh) [inline src/Symfony/Component/Mailer/CHANGELOG.md]: What's the reason for this change?

Comment (wouterj) [reply src/Symfony/Component/Mailer/CHANGELOG.md]: I think it's a good practice to use a domain that is reserved by IANA for documentation purposes, instead of a domain that could potentially be registered for malicious use in the future.
I'm aware this makes the example a little less explicit (no unicode in the domain part anymore), but I think it shows the meaning enough already.

Comment (xabbuh) [reply src/Symfony/Component/Mailer/CHANGELOG.md]: ah, works for me 👍

Comment (xabbuh) [reply src/Symfony/Component/Mailer/CHANGELOG.md]: ah, works for me 👍

File: UPGRADE-7.2.md [hunk 3]
```diff
@@ -8,10 +8,40 @@ Read more about this in the [Symfony documentation](https://symfony.com/doc/7.2/
...
+ * [Serializer](#Serializer)
+ * [Translation](#Translation)
+ * [Webhook](#Webhook)
+ * [Yaml](#Yaml)
+
 Cache
 -----
 
- * `igbinary_serialize()` is not used by default when the igbinary extension is installed
+ * [BC BREAK] `igbinary_serialize()` is no longer used instead of `serialize()` when the igbinary extension is installed,
```
Comment (fabpot) [inline UPGRADE-7.2.md]: Why is it a BC break?

File: src/Symfony/Component/Mailer/CHANGELOG.md (hunk 2 above)
Comment (fabpot) [reply src/Symfony/Component/Mailer/CHANGELOG.md]: I would use `dømi.example` as per RFC 2606 which recommends using the `.example` TLD for documentation.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: UPGRADE-7.2.md (hunk 3 above)
Comment (wouterj) [reply UPGRADE-7.2.md]: Thanks for bringing this up!

I was reading https://github.com/symfony/symfony/pull/57702#issuecomment-2241598355 As far as I understood, people might have been installing igbinary for cache performance reasons. In 7.2, there is no way - as a framework user - to enable igbinary usage again. These applications are affected by a performance penalty, even if they were not at risk with the behavioral incompatibility.

Maybe BC break is too harsh of a classification, but this does sound more than just "oh something is deprecated" to me.

Comment (fabpot) [reply UPGRADE-7.2.md]: Maybe

```suggestion
 * `igbinary_serialize()` is no longer used instead of `serialize()` when the igbinary extension is installed (performance might be impacted),
```

File: UPGRADE-7.2.md [hunk 4]
```diff
@@ -24,7 +54,27 @@ Console
...
+   ```
+
+   *After*
+   ```yaml
+   services:
+       App\Handler:
+           tags: ['app.handler']
+
+   App\HandlerCollection:
+       arguments: [!tagged_iterator app.handler]
```
Comment (MrYamous) [inline UPGRADE-7.2.md]: Indentation seems inexact here
```suggestion
       App\HandlerCollection:
           arguments: [!tagged_iterator app.handler]
```

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @wouterj.

**Comment by OskarStark** [comment]
Thank you very much @wouterj that is very key to be able to upgrade smoothly. Thanks for your patience 🥰
